test/integration/functional_test.go

```diff
@@ -31,7 +33,13 @@ func TestFunctional(t *testing.T) {
 		Args:       *args,
 		T:          t}
 	minikubeRunner.EnsureRunning()
-
+	integrationTestImages := []string{"busybox:glibc"}
+	if err := machine.CacheImages(integrationTestImages, constants.ImageCacheDir); err != nil {
+		t.Fatalf("caching images: %s", err)
+	}
+	if err := machine.LoadFromCacheBlocking(&minikubeRunner, constants.ImageCacheDir); err != nil {
```

Comment:
We don't have this in a loop. Is that just because we only have one image here? It's used in a loop later on.

Reply:
Yeah we're just calling the single loader directly for busybox, where the other one loads a list of images.